services/api/mw2pdf/classes/PdfGenerator.js

```diff
@@ -9,6 +11,9 @@ import {
 } from 'pdf-lib'
 import { Pdf } from './Pdf.js'
 
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
Just because I feel the need to make _some_ comment, this _could_ be simplified into one line:

`const __dirname = path.dirname(fileURLToPath(import.meta.url));`

but it's by no means necessary!

Reply:
Appreciated!

I actually prefer to optimize around code legibility rather than minimizing lines anyway so I'll keep it as is.  Plus now we have a handy __filename variable which I'm *SURE* will be useful...  😂